tfx/components/example_gen/write_split.py

```diff
@@ -77,13 +77,21 @@ def WriteSplit(
                   schema,
                   file_name_suffix='.parquet',
                   codec='snappy'))
+    
+    custom_config = exec_properties.get('custom_config')
+    custom_config = eval(custom_config) if isinstance(custom_config, str)
```

Comment:
We discourage using `eval`, because it can create security vulnerabilities. 
Consider using `ast.literal_eval` for safely evaluating strings containing Python expressions

Reply:
thank you for the feedback -- implemented proposed solution